scripts/supplyTokens.ts

```diff
@@ -0,0 +1,25 @@
+import { ethers } from "hardhat";
+
+async function main() {
+  const tokenAaddr = "<from deploy logs>";
+  const tokenBaddr = "<from deploy logs>";
+  const metamaskAddress = "<address to sent to>";
+  
+  const [owner] = await ethers.getSigners();
+  await owner.sendTransaction({
+    to: metamaskAddress,
+    value: ethers.utils.parseEther("100.0"), // Sends exactly 100.0 ether
+  });
+  
+  const tokenA = await ethers.getContractAt("TestERC20", tokenAaddr);
+  const tokenB = await ethers.getContractAt("TestERC20", tokenBaddr);
+  await tokenA.transfer(metamaskAddress, ethers.utils.parseEther("100.0"));
+  await tokenB.transfer(metamaskAddress, ethers.utils.parseEther("100.0"));
+}
+
```

Comment:
why is this file needed? can't we generate fake addresses with ether using `npx hardhat node` and then import those accounts in metamask to use?

Reply:
You are right, removed this file, and updated readme.